Tests/kaas/plugin/plugin_kind.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+import os
+import os.path
+from interface import KubernetesClusterPlugin
+from pytest_kind import KindCluster
+import logging
+logger = logging.getLogger("plugin_kind")
+
+
+class PluginKind(KubernetesClusterPlugin):
+    """
+    Plugin to handle the provisioning of kubernetes cluster for
+    conformance testing purpose with the use of Kind
+    """
+    def __init__(self, config_file=None):
+        logger.info(f"Init provider plug-in of type {self.__class__.__name__}")
```

Comment:
The string interpolation doesn't make much sense here, because the class is known here.
We can either move this to `run_plugin.init_plugin` or (even simpler) just put in `PluginKind` here.

Reply:
Yes good catch I switched it to `PluginKind`